Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
Traceback (most recent call last):
  File "/home/maksandr2003/telegram-bot/main.py", line 7, in <module>
    import nest_asyncio
ModuleNotFoundError: No module named 'nest_asyncio'
